src/index.js

```diff
@@ -108,32 +106,24 @@ function ncc (
     existingAssetNames.push(`${filename}.cache`);
     existingAssetNames.push(`${filename}.cache${ext}`);
   }
+
+  let tsconfig = {};
   const resolvePlugins = [];
-  // add TsconfigPathsPlugin to support `paths` resolution in tsconfig
-  // we need to catch here because the plugin will
-  // error if there's no tsconfig in the working directory
-  let fullTsconfig = {};
+  const resolveModules = [];
+  const compilerOptions = tsconfig.compilerOptions || {};
```

Comment:
What does this line do? Why assign `compilerOptions` before `tsconfig.json` is even read?

Reply:
probably straightforward mistake? 😓

Reply:
The latest commit resolves the problem. The review can be marked as resolved now.